gemini/sample-apps/finance-advisor-spanner/requirements.txt

```diff
@@ -0,0 +1,11 @@
+streamlit
+google-cloud-language>=2.9.1
```

Comment:
Is this library actually used?

Reply:
this is the one that has spanner inside.

Reply:
Why is spanner in the language library?